NuGetPackageVerifier.json

```diff
@@ -0,0 +1,134 @@
+{
+    "adx": { // Packages written by the ADX team and that ship on NuGet.org
+        "rules": [
+            "AssemblyHasDocumentFileRule",
+            "AssemblyHasVersionAttributesRule",
+            "AssemblyHasServicingAttributeRule",
+            "SatellitePackageRule",
+            "StrictSemanticVersionValidationRule"
+        ],
+        "packages": {
+            "dnx-clr-win-x64": {
+                "VERSION_INFORMATIONALVERSION": {
+                    "bin\\lib\\Microsoft.Dnx.DesignTimeHost\\Newtonsoft.Json.dll": "3rd party DLLs don't need this",
+                    "bin\\lib\\Microsoft.Dnx.Tooling\\Newtonsoft.Json.dll": "3rd party DLLs don't need this"
+                },
+                "SERVICING_ATTRIBUTE": {
+                    "bin\\lib\\Microsoft.Dnx.DesignTimeHost\\Microsoft.Framework.NotNullAttribute.Sources.dll": "BUG: This file shouldn't be here",
+                    "bin\\lib\\Microsoft.Dnx.DesignTimeHost\\Newtonsoft.Json.dll": "3rd party DLLs don't need this",
+                    "bin\\lib\\Microsoft.Dnx.Tooling\\Microsoft.Framework.NotNullAttribute.Sources.dll": "BUG: This file shouldn't be here",
```

Comment:
I looked into a few .Sources packages, and this seems to be true.  I filed bugs on Common and Dnx repos to fix this:

https://github.com/aspnet/Common/issues/41
https://github.com/aspnet/dnx/issues/2720

Other repos also have some .Sources packages so we'll need to identify each and fix this.